R/readIEA_coolingEfficiencies.R

```diff
@@ -0,0 +1,21 @@
+#' Read IEA Global Weighted Average of Air Conditioning Efficiencies
+#'
+#' @author Hagen Tockhorn
+#'
+#' @importFrom dplyr mutate
+#' @importFrom quitte as.quitte
+#' @importFrom magclass as.magpie
+
+readIEA_coolingEfficiencies <- function() { #nolint object_name_linter
```

Comment:
I know this works. But the official syntax is like this:
```suggestion
readIEA_coolingEfficiencies <- function() { # nolint: object_name_linter.
```